tests/src/integration/tests/test_auth.cpp

```diff
@@ -109,7 +109,7 @@ CASSANDRA_INTEGRATION_TEST_F(AuthenticationTests, InvalidEmptyCredentials) {
   CHECK_FAILURE;
 
   // Iterate over all known/supported protocol versions
-  logger_.add_critera("Key may not be empty");
+  logger_.add_critera("Password must not be null");
```

Comment:
Updated to match the message observed for the default C* version (3.11.6)

Reply:
Maybe leave the old criteria and add the new one? Then update the logger count check to a `> 0`? I don't know why this asserts a specific count...

Reply:
Yeah, I like that quite a bit better than my initial change.  I'll update.